src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $main-card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The transform scale on hover might make the content around the card jump or get pushed aside which can be disruptive to the layout. Consider using a shadow or a slight change in color to indicate hover instead. This would also align with the checklist's focus on smooth visual effects that don't disrupt the layout.